server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +647,35 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to room from broadcast-signal endpoint
+			if (collborationSessionEventEmitter !== undefined) {
+				collborationSessionEventEmitter.on("broadcastSignal", (broadcastSignal: ICollaborationSessionEvent) => {
+					const tenantId = broadcastSignal.tenantId;
+					const documentId = broadcastSignal.documentId;
+					const roomToBroadcastSignal: IRoom = { tenantId, documentId };
+
+					// No-op if the room (collab session) that signal came in from is different
+					// than the current room. We reuse websockets so there could be multiple rooms
+					// that we are sending the signal to, and we don't want to do that.
+					if (roomToBroadcastSignal === room) {
```

Comment:
Need to check the properties one by one; this object comparison (by reference) will always return `false`.